jina/__init__.py

```diff
@@ -140,7 +141,7 @@ def import_classes(namespace: str, targets=None,
     else:
         raise TypeError(f'target must be a set, but received {targets!r}')
 
-    depend_tree = {}
+    depend_tree: Dict[str, str] = {}
```

Comment:
no need. for this particular file, speed is the first priority

Reply:
Thanks for your response.
Remove the usages of the import in a fixup commit (will rebase and force push later).